tests/data/runFull/config.json

```diff
@@ -33,7 +33,7 @@
             "type": "nvarchar",
             "size": "255",
             "nullable": false,
-            "default": null
+            "default": "some-default-value"
```

Comment:
Nasimuloval som problem z ticketu (tam sa to tyka `decimal` a default `0.0`), ale ten problem je celkovo.

![image](https://user-images.githubusercontent.com/19371734/204777398-2f3fe404-91d0-490a-b28c-3c4520d7adc6.png)

Myslim, ze tam chyba `DEFAULT` keyword + string hodnoty by sa mali nejak escapovat.